packages/dds/tree/src/domains/json/jsonCursorLegacy.ts

```diff
@@ -0,0 +1,227 @@
+/*!
```

Comment:
I'd prefer if this file move was not in the same PR as the implementation logic so the diff was better.

Anyway, is it safe to assume this file was just moved, and not edited?

Reply:
Yup, I made a note about it in the description. I can rename the new one if you think that would help, assuming that Github would detect the file rename as long as its old name isn't still being used.

Reply:
You can leave it as is. Delta between the old and new implementations ended up being useful anyway.